service-control/src/main/java/fi/nls/oskari/control/ActionParameters.java

```diff
@@ -150,6 +150,22 @@ public double getRequiredParamDouble(final String key) throws ActionParamsExcept
             throw new ActionParamsException(errMsg);
         }
     }
+    /**
+     * Returns a cleaned up (think XSS) value for the requested parameter
+     * @param key parameter name
+     * @return cleaned up value for the parameter as boolean
+     * @throws ActionParamsException if parameter is not found, is empty or can't be parsed as double
+     */
+    public boolean getRequiredParamBoolean(final String key) throws ActionParamsException {
+        final String errMsg = "Required parameter '" + key + "' missing!";
```

Comment:
You don't need to give this string (even as other similar do) since it's the default message for getRequiredParam(key).

Reply:
Fixed